src/Symfony/Bundle/SecurityBundle/DependencyInjection/Compiler/AddStrategyResolversPass.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\SecurityBundle\DependencyInjection\Compiler;
+
+use Symfony\Component\DependencyInjection\Reference;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+
+class AddStrategyResolversPass implements CompilerPassInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function process(ContainerBuilder $container)
+    {
+        if (!$container->hasDefinition('security.access.decision_manager')) {
+            return;
+        }
+
+        $strategyResolvers = new \SplPriorityQueue();
```

Comment:
Oh sorry I didn't know new feature have to be on master, I only did patch PR until now

I rebased my branch and realized there is still some work that needs to be done because of the `TraceableAccessDecisionManager` (the fact that the strategy is not unique now).

Unrelated but about your comment on the `PriorityTaggedServiceTrait` trait : I actually almost copy / pasted the `AddSecurityVotersPass` when I did mine. I just checked and this one is not using the trait... Should it be done in a separate PR as there might be problems of order.